<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormReport</class>
 <widget class="QWidget" name="FormReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>С</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditBeg">
       <property name="displayFormat">
        <string>dd.MM.yy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>По</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEditEnd">
       <property name="displayFormat">
        <string>dd.MM.yy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUpd">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonPart">
       <property name="text">
        <string>По партиям</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMark">
       <property name="text">
        <string>По маркам</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>olap/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
